src/components/common/footer/Footer.jsx

```diff
@@ -1,8 +1,44 @@
 import React from 'react';
 import './footer.scss';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faFacebookF, faLinkedinIn, faTwitter, faMediumM } from "@fortawesome/free-brands-svg-icons";
 
```

Comment:
This is erroring out for me. I think it's supposed to be ``@fortawesome/free-solid-svg-icons``

Reply:
Yup. Looks like it.

Reply:
Yeah the normal svg-icons didn’t have the the icons needed that’s why I had to npm install the brand-icons one. It should be in the package.json

Reply:
Nevermind. I see now. It wasn't in package.json so it didn't install. I don't see package.json on here so it didn't get checked in.

Reply:
Ohh okay I should have added that in the commit comments. Does it work now though?

Reply:
When I pulled, it looks like it works.